org.psi_im.Psi.json

```diff
@@ -0,0 +1,164 @@
+{
+  "id": "org.psi_im.Psi",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "5.15-21.08",
+  "base": "io.qt.qtwebengine.BaseApp",
+  "base-version": "5.15-21.08",
+  "sdk": "org.kde.Sdk",
+  "command": "psi",
+  "rename-icon": "psi",
+  "rename-desktop-file": "psi.desktop",
+  "rename-appdata-file": "psi.appdata.xml",
+  "finish-args": [
+    "--device=dri",
+    "--share=ipc",
+    "--share=network",
+    "--socket=wayland",
+    "--socket=fallback-x11",
+    "--socket=pulseaudio",
+    "--talk-name=org.kde.StatusNotifierWatcher",
+    "--talk-name=org.freedesktop.Notifications",
+    "--own-name=org.psi-im.Psi",
```

Comment:
you should submit a patch upstream to have them use a valid dbus name.

it should have been part of the appdata patch

Reply:
https://github.com/psi-im/psi/search?q=org.psi-im.Psi&type=code Can't find it. Maybe it is already fixed.

Reply:
or maybe it's not used at all.

Reply:
Please drop it, then.

Reply:
No, it is required; otherwise the application does not start. Which also makes me wonder: if this is invalid and forbidden according to DBus specifications that are enforced here, why does it work at all?